content/en/monitoring/notification-manager.md

```diff
@@ -388,7 +390,7 @@ spec:
 EOF
 ```
 
-Slack receiver labeled with `type: tenant` only receive notification happend in the namespace which user be in. If you want to receive notification without namespace, you can create a global slack receiver labeled with `type: global` like this.
+Slack receiver labeled with `type: tenant` only receive notification happened in the namespace which user be in. If you want to receive notification without namespace, you can create a global slack receiver labeled with `type: global` like this.
```

Comment:
Slack receiver labeled with `type: tenant` only receive notification happened in the namespace which user be in => Slack receiver labeled with `type: tenant` only receive notification happened in the namespace where the users are  invited

Is this description more exact?

Reply:
@Sherlock113 Please help me to verify if my comment much is more exact here.

Reply:
To my understanding,

1. "If you want to receive notification without namespace": this is ambiguous to me. My guess is that you mean you want users to receive notifications no matter what namespace they are in.

2. "slack receiver" is not a person here. Rather, it receives notifications and send them to the actual user (person).

Based on the above, here is my version:

The slack receiver labeled with `type: tenant` only receives notifications that happen in the namespace where the tenant has already existed. If you want them to receive notifications regardless of the namespace, you can create a global slack receiver labeled with `type: global` as follows:

Reply:
@wanjunlei Please take a look at @Sherlock113 's comment, I agree with this one.